### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by Amazinggracee**
### Project requirements

- [ ] Create a new app. Give it a meaningful name.
- [ ] Initialize your project with Git.
- [ ] Make sure that your project has Postgres database set up.
- [ ] Build your project schema.
- [ ] Create and run the necessary migration files.
- [ ] Table and column names should match the ERD diagram.
note: photo for users table should be a link to an image
- [ ] Foreign keys should be included.
- [ ] Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
- [ ] All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Review by joel-kalema — COMMENTED** [review]
Hi @Amazinggracee 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request  🛑 

pirates-of-the-caribbean-johnny-depp


- [ ] You have submitted a project that doesn't have a description [README](https://github.com/microverseinc/readme-template)


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+Project architecture and requirments provided by Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (fobadara) [inline README.md]: [REQUIRED] Your license link currently takes me to a 404 page. Kindly fix this to make your project more professional.

File: db/migrate/20230815163624_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :photo
+
+      t.timestamps
+    end
```
Comment (fobadara) [inline db/migrate/20230815163624_create_users.rb]: [REQUIRED] Kindly include bio and posts_counter just as it is in the ERD diagram

File: db/migrate/20230815163650_create_posts.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :content
+      t.references :author, foreign_key: { to_table: :users }  # Use :users, not :authors
+      t.timestamps
+    end
+  end
+end
```
Comment (fobadara) [inline db/migrate/20230815163650_create_posts.rb]: [REQUIRED] Kindly include text, comments_counter and likes_counter

File: db/migrate/20230815173748_create_comments.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :content
```
Comment (fobadara) [inline db/migrate/20230815173748_create_comments.rb]: [REQUIRED] Kindly use text instead of content just like the ERD diagram

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Amazinggracee, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

REDO

## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230815173748_create_comments.rb (hunk 4 above)
Comment (Amazinggracee) [reply db/migrate/20230815173748_create_comments.rb]: I have changed the content to text.

File: README.md (hunk 1 above)
Comment (Amazinggracee) [reply README.md]: So, sorry, I have linked the right file. Thank you so much.

File: db/migrate/20230815163624_create_users.rb (hunk 2 above)
Comment (Amazinggracee) [reply db/migrate/20230815163624_create_users.rb]: I have added the bio and the counter. All is up to date now.

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Amazinggracee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] All migrations run without error :white_check_mark: 
- [x] Database schema is set up correctly :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark: 
- [x] Pull request has a descriptive pull request title and summary :white_check_mark:    

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**